src/main/java/com/liferay/faces/util/render/BufferedScriptResponseWriter.java

```diff
@@ -37,11 +37,12 @@
 	// Logger
 	private static final Logger logger = LoggerFactory.getLogger(BufferedScriptResponseWriter.class);
 
-	StringWriter stringWriter = null;
+	// Package-Private Data Members
+	@Deprecated
+	/* package-private */ StringWriter stringWriter = new StringWriter();
 
-	public BufferedScriptResponseWriter() {
-		this.stringWriter = new StringWriter();
-	}
+	// Private Final Data Members
+	private final StringWriter _stringWriter = new StringWriter();
```

Comment:
@stiemannkj1 As I recall, usage of the `final` keyword for private data members is not commonly used in the codebase.